llvm/include/llvm/Analysis/LoopAccessAnalysis.h

```diff
@@ -37,6 +37,8 @@ class Value;
 struct VectorizerParams {
   /// Maximum SIMD width.
   static const unsigned MaxVectorWidth;
+  /// Maximum LMUL factor.
```

Comment:
LMUL is a RISCV specific term, would be good to frame in a general term if possible?

Reply:
+1, how about UF?
Note that UF in general may be a non-power-of-2, whereas LMUL is, iiuc.